src/js/__tests__/gamestate-test.js

```diff
@@ -66,7 +68,7 @@ describe("GameState", function() {
     state.makeMove({op: "refreshPlayers"})
 
     // get a card that implements kill
-    state.draw({"name":"eve"}, {"kill":true, "cost":0})    
+    state.draw({"name":"eve"}, {"kill":true, "cost":0, "target":TARGETS.OPPONENT_PERMANENT, randomTarget:true})    
```

Comment:
just a style nitpick, now the keys for this object are inconsistently formatted. how about, when it's a literal hash just always using the

{name: "eve"}

syntax (like you do for randomTarget here)